src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/DropColumns.cs

```diff
@@ -5,26 +5,27 @@
 using Microsoft.PowerFx.Core.App.ErrorContainers;
 using Microsoft.PowerFx.Core.Binding;
 using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Functions;
 using Microsoft.PowerFx.Core.Localization;
 using Microsoft.PowerFx.Core.Types;
 using Microsoft.PowerFx.Core.Utils;
 using Microsoft.PowerFx.Syntax;
 
-#pragma warning disable SA1402 // File may only contain a single type
-#pragma warning disable SA1649 // File name should match first type name
-
 namespace Microsoft.PowerFx.Core.Texl.Builtins
 {
     // DropColumns(source:*[...], name:s, name:s, ...)
     internal sealed class DropColumnsFunction : FunctionWithTableInput
```

Comment:
>DropColumnsFunction [](http://example.com/codeflow?start=26&length=19)

Why not update ShowColumns as well?

Reply:
I considered it out of scope for this PR 
We can address it in a future PR for sure.

Reply:
Ditto - future PR. This is laying a foundation, and it would even enabling doing the other functions in parallel - similar to what Bryan did for migrating to CheckType